baseplate/metrics.py

```diff
@@ -193,8 +194,31 @@ def __exit__(self, exc_type, value, traceback):
 
     def flush(self):
         """Immediately send the batched metrics."""
+        for name, counter in self.counters.items():
+            counter.send()
         self.transport.flush()
 
+    def counter(self, name):
+        """Return a BatchCounter with the given name.
+
+        BatchCounter instances are stored in the :term:`counters` attribute.
```

Comment:
This is an internal implementation detail and probably shouldn't be exposed in the documentation.